sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/models/PathExpiryMode.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.storage.file.datalake.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import java.util.Collection;
+
+/** Defines values for PathExpiryMode. */
+public final class PathExpiryMode extends ExpandableStringEnum<PathExpiryMode> {
```

Comment:
I don't think we need this class, see my previous comment.

Reply:
What does `FileScheduleDeletionOptions` have for this property today?  I'm unable to find this class in the repo.  Because it appears we are adding `PathExpiryMode` now, and ScheduleForDeletion, I think this class is unnecessary.